astro/src/content/docs/get-started/core-concepts/roles.mdx

```diff
@@ -149,7 +149,6 @@ _FusionAuth user_support_manager_
 | user         | Edit a user, except for any identity information that could be used to authenticate. For example, the email and username cannot be modified.                                    |
 | user         | Lock a user account.                                                                                                                                                            |
 | user         | Unlock a user account.                                                                                                                                                          |
-| user         | Modify 2FA settings if available.                                                                                                                                               |
```

Comment:
I think this is accurate

However, I pulled up the local app to test this and while you cannot modify a user's 2FA settings you can see them so maybe a better edit would be "View 2FA methods if available"

As an aside when I tried to send the verification code when clicking the "delete" button no email was sent. There may be a bug there.

Reply:
I submitted an issue for that. https://github.com/FusionAuth/fusionauth-issues/issues/2698

Learned it does the same thing for email (and I presume SMS but didn't have an SMS provider set up in my environment).

Reply:
I can do "View 2FA methods if available." instead also.